src/views/pages/globalping/_index.html

```diff
@@ -2158,65 +2121,65 @@
 				infoWindows.forEach(iw => iw.close());
+
+				this.despiderifyMapMarkers(mapMarkers);
 			});
 		},
 		updateClusters (probes) {
-			let svg = window.btoa(`<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-				<g filter="url(#filter0_d_6106_3045)">
-				<circle cx="12" cy="10" r="6" fill="#17D4A7"/>
-				<circle cx="12" cy="10" r="7" stroke="white" stroke-width="2"/>
-				</g>
-				<defs>
-				<filter id="filter0_d_6106_3045" x="0" y="0" width="24" height="24" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
-				<feFlood flood-opacity="0" result="BackgroundImageFix"/>
-				<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
-				<feOffset dy="2"/>
-				<feGaussianBlur stdDeviation="2"/>
-				<feComposite in2="hardAlpha" operator="out"/>
-				<feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.2 0"/>
-				<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow_6106_3045"/>
-				<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow_6106_3045" result="shape"/>
-				</filter>
-				</defs>
-				</svg>`);
-
-			probes.forEach((probe) => {
-				let probeInfoWindow = new google.maps.InfoWindow({
-					content: `<div class="gp-map_popup">
-							<div class="gp-map_popup_descr">${probe.location.network}</div>
-							<div class="gp-map_popup_descr">(${probe.location.city}, ${probe.location.country})</div>
-						</div>`,
-				});
+			let preparedMarkers = Object.keys(probes).map((key) => {
+				let [ lat, lng ] = key.split(', ');
+				let marker = probesMarkers.find(pm => pm.position.lat === lat && pm.position.lng === lng);
 
-				let probeMarker = new google.maps.Marker({
-					map,
-					icon: { url: `data:image/svg+xml;base64,${svg}` },
-					position: { lat: probe.location.latitude, lng: probe.location.longitude },
-					optimized: false,
-				});
+				if (!marker) {
+					marker = this.createMapClusterMarkerWithIW(key, probes[key]);
+					probesMarkers.push(marker);
+				}
 
-				google.maps.event.addListener(probeMarker, 'click', () => {
-					probesInfoWindows.forEach(piw => piw.close());
-					probeInfoWindow.open(map, probeMarker);
-				});
+				marker.map = map;
 
-				probesInfoWindows.push(probeInfoWindow);
-				probesMarkers.push(probeMarker);
+				return marker;
 			});
 
 			// eslint-disable-next-line no-undef
 			let probeCluster = new markerClusterer.MarkerClusterer({
-				markers: probesMarkers,
+				markers: preparedMarkers,
 				map,
 				renderer: {
-					render: ({ markers, _position: position }) => {
-						return new google.maps.Marker({
-							position: {
-								lat: position.lat(),
-								lng: position.lng(),
-							},
-							icon: `${this.get('@shared.assetsHost')}/img/globalping/map_cluster.svg`,
-							label: { text: String(markers.length), color: '#fff', fontSize: '16px', fontWeight: '600' },
+					render ({ markers, position }) {
```

Comment:
The text doesn't seem to be vertically centered: https://dl.dropboxusercontent.com/scl/fi/jqnhkie2htwrnfenx26xb/2024-12-03_15-52-34.png?rlkey=xevkh5v8aoo3ptnpucucaiuol&dl=0

Reply:
fixed